plugins/LegacyProfileReader/DictionaryOfDoom.json

```diff
@@ -0,0 +1,75 @@
+{
+    "source_version": "15.04",
+    "target_version": 1,
```

Comment:
No, it's actually the profile version, which is still at 1, not the API version. The `source_version` is not used at the moment by the way.